plugins/reporters/spdx/src/main/kotlin/SpdxDocumentReporter.kt

```diff
@@ -174,10 +174,9 @@ private fun ObjectNode.patchSpdx23To22(): ObjectNode {
         val extRefs = pkg.get("externalRefs") ?: return@forEach
 
         extRefs.forEach { ref ->
-            val refCat = ref.get("referenceCategory") ?: return@forEach
+            val refCatText = ref.get("referenceCategory")?.textValue() ?: return@forEach
```

Comment:
While coding this, I deliberately kept it that way to make this line more similar to line 174. But I'm also ok with the change, now that you did it.

Reply:
Ok, the next commit changes line 174 anyway...